OneOneOne/OneOneOne/tests/test_message_handler.py

```diff
@@ -0,0 +1,154 @@
+import unittest
+from pathlib import Path
+from builder.pystache_message_builder import PystacheMessageBuilder
+from utilities.file_utilities import FileUtilities
+from utilities.xml_utilities import XmlUtilities
+from OneOneOne.OneOneOne.message_handler import MessageHandler
+
+from definitions import ROOT_DIR
+
+
+class MessageHandlerTest(unittest.TestCase):
+    expectedXmlFileDir = Path(ROOT_DIR) / 'OneOneOne' / 'tests' / 'expected_output_xmls'
+    inputXmlFileDir = Path(ROOT_DIR) / 'OneOneOne' / 'tests' / 'input_xmls'
+
+    def test_action_not_matching_service(self):
+        builder = PystacheMessageBuilder(str(self.inputXmlFileDir), 'action_does_not_match_service')
+
+        with self.subTest("Two differing services result in a 500 error"):
+            service_dict = {'action': "urn:nhs-itk:services:201005:SendNHS111Report-v2-0-ThisDoesNotMatchBelow",
+                            'service': "urn:nhs-itk:services:201005:SendNHS111Report-Bad_Service-ThisDoesNotMatchAbove"}
+
+            msg = builder.build_message(service_dict)
+            mh = MessageHandler(msg)
+            status_code, response = mh.check_action_types()
+
+            assert (status_code == 500)
```

Comment:
This should be using `self.assertEqual` (i.e the unittest methods) shouldn't it?

Reply:
Updated the values to use `self.assertEqual/False`